tests/units/Ticket.php

```diff
@@ -990,21 +984,22 @@ public function testFormPostOnly() {
    }
 
    public function testFormTech() {
-      $auth = new \Auth();
-      $this->boolean((boolean)$auth->Login('tech', 'tech', true))->isTrue();
+      global $DB;
 
       //create a new ticket
       $ticket = new \Ticket();
       $this->integer(
          (int)$ticket->add([
-            'name'         => '',
-            'description'  => 'A ticket to check displayed tech form',
-            'content'      => ''
+            'name'                => '',
+            'content'             => 'A ticket to check displayed tech form',
+            '_users_id_requester' => '3', // post-only
+            '_users_id_assign'    => '4', // tech
```

Comment:
Better to get user id from name rather than use an hardcoded id (my change in the future)

Reply:
Hum... Nevermind, ids are already used in other places in this test; let as it is OK.